### ISSUE TITLE
[FVT] docker :def command does not work for docker instance definition

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#601]
Test OS: MN and docker host is ubuntu 14.04
Test xCAT version: 
```
root@c910f04x30v11:~# lsxcatd -v
Version 2.12 (git commit 2e8f66ff1fae273cdbdad0a47743121ca00bef74, built Thu Jan 14 19:32:28 EST 2016)
```

Debug information: xxx
```
root@c910f04x30v11:~# chdef hostsc1   objtype=node dockerflag=15.04 dockerhost=c910f04x30v11 dockercpus=2 dockermemory=4096 dockernics=en0 groups=docker ip=10.0.0.111  mgt=docker
Error: 'objtype' is not a valid attribute name for an object type of 'node'.
Error: Skipping to the next attribute.
Error: 'dockernics' is not a valid attribute name for an object type of 'node'.
Error: Skipping to the next attribute.
--------------------> then def command hang 
```

**Comment by zet809** [comment]
Since the docker instance networking functions are not implemented in this sprint, the dockernics attribute can not be used at present. I have modified mini-design.
And for objtype, it is invalid for *def command. 
So, close this defect.

**Comment by junxiawang** [comment]
But the command chdef  hang ,I do not think this should be hanged here.

**Comment by junxiawang** [comment]
chdef host1c1 dockerflag=15.10 dockerhost=c910f04x30v11 dockercpus=2 dockermemory=4096 groups=docker  mgt=docker
-----------------> using this command still hang and no output.

**Comment by junxiawang** [comment]
root@c910f04x30v11:~# rmdef -t node ttt
Error: Could not find an object named 'ttt' of type 'node'.
----->will hang here

**Comment by daniceexi** [comment]
@zet809 This issue caused by a topy in the Schema.pm. Please change the 'ndoe' to 'node' in L2579

~~~~
2576                 {attr_name => 'dockerflag',
2577                  only_if => 'mgt=docker',
2578                  tabentry => 'vm.othersettings',
2579                  access_tabentry => 'vm.ndoe=attr:node',
2580                 },
~~~~

**Comment by zet809** [comment]
Fixed in pull request #611 .

**Comment by junxiawang** [comment]
Tested using latest xcat build lsdef - Version 2.12
